### PR TITLE
Update suggestion

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
Description:

- I don't think the `ElevationRequirement` should be added. (This was also not be added in the previous version), so I removed it.
( More: [In version 1.3](https://github.com/microsoft/winget-pkgs/pull/173045#issuecomment-2347086572) )

- ~~This package is **available on x64,x86,arm, and arm64**, and I added them.~~ (Resolved)

- I updated the **tags** in the Chinese (zh-CN) manifest.

- I didn't add `ProductCode` to [the iss file for this package](https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/pack.iss) , so I removed the `ProductCode` from the manifest.

- ~~I have a _question_ , does winget automatically find the appropriate silent install switch when the manifest is not explicitly specified?~~ (Resolved)

- The License is **GNU General Public License v3.0** . No more. (软著登记号是之前版本的)

- I added the `ReleaseNotes` for this version.

> [!WARNING]
> In previous versions, the version number was preceded by a "v", but this version does not. Does this cause Upgrade-Always-Available ?

### CODE REVIEW

File: manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Created with komac v2.6.0
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-3958.Win32NT
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
 
 PackageIdentifier: DuckStudio.GitHubLabelsManager
-PackageVersion: '1.5'
+PackageVersion: 1.5
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.yaml:5]:
```suggestion
PackageVersion: '1.5'
```

**Comment by DuckDuckStudio** [comment]
Changes done.
感谢您的更新，麻烦您了。

---

我似乎无法强推掉合并主分支的提交
image

File: manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  > [!TIP]
+  > 本版本为修复 (Fix)
+
+  ## 修复 (Fix)
+  - [x] fix: 修正 422 与 401 错误返回 (Fix error return of 422 and 401 status codes.)
+
+  ---
+
+  **Full Changelog**: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/compare/v1.4...v1.5
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/Chinese_git/releases/tag/v1.5
```
Comment (Dragon1573) [inline manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.locale.en-US.yaml]:
```suggestion
ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/tag/v1.5
```

Is it another "accident"? 😄

**Review by Dragon1573 — CHANGES_REQUESTED** [review]
I previously create the manifest with a brand new package identifier accidentally, everything were inspected by _Komac_, including `ElevationRequirement`, `ProductCode` and `AppsAndFeaturesEntries`.

File: manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.locale.en-US.yaml (hunk 2 above)
Comment (DuckDuckStudio) [reply manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.locale.en-US.yaml]: Thank you for catching this issue!

**Comment by Dragon1573** [comment]
> [!CAUTION]
>
> Unwanted commits are included in the pull requests!
>
> Since fixing them are too complicated. This pull request will be rejected 😞

image

> [!NOTE]
>
> The related pull request microsoft#195394 will also be closed. I'll try to recreate a new pull request of this package.

**Pull request closed without merging**